WikiWatershed/model-my-watershed
# BiG-CZ Map Popups

## Overview
Adds popovers to the BiG-CZ results we display on the map (Currently CINERGI and WDC). 


**Wireframe**
![screen shot 2017-08-22 at 11 18 33 am](https://user-images.githubusercontent.com/7633670/29572939-a9d14814-872b-11e7-9702-d4a306b9753d.png)


Connects #2136 

### Demo

![screen shot 2017-08-22 at 11 10 47 am](https://user-images.githubusercontent.com/7633670/29572913-96dde3e8-872b-11e7-93d5-3463ee1ce369.png)
![screen shot 2017-08-22 at 11 11 20 am](https://user-images.githubusercontent.com/7633670/29572915-97185c26-872b-11e7-9285-b8c182692fbf.png)

![th8cajmzhy](https://user-images.githubusercontent.com/7633670/29572967-b602621c-872b-11e7-98b8-b867a67c766f.gif)


### Notes
1. The wireframes don't reflect the actual data for the catalogs, so I just reused the `ResultView` for the popups while we determine what they should contain
2. The interaction for opening a CINERGI popup is weird because you have to click the skinny boundary line. Clicking the inside of the boundaries would also be bad though; boundaries overlap.
3. I didn't match the wireframe "detail" button exactly because setting it to `ui-secondary` seemed much to dark on the white background
3. Refactored the enter-detail-mode triggering to allow the popup access

## Testing Instructions

 * Pull and `bundle`. There are style changes, make sure the sass compiles.
 * Get results for the three tabs by searching something like `water` 
    - Confirm you can still open the details from the sidebar from all three catalogs
    - Confirm if you click on a Cinergi map boundary, its corresponding popup opens and you can use its details button
    - Confirm the same for the WDC points
